go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/iamseth/oracledb_exporter
+module github.com/observiq/oracledb_exporter
```

Comment:
If you use a replace directive in the go.mod where this is used, this shouldn't be necessary to do, that way you could use this branch to open a PR upstream instead of having to make a separate one.

Reply:
This was mostly because of the work I did where it is required in an import (i.e. not from the iamseth remote) or the commit won't be found. Will change back once we want to contribute upstream

Reply:
I think you could if the collector package existed but it doesn't in this case 😬